src/doc/src/reference/manifest.md

```diff
@@ -11,31 +11,31 @@ Every manifest file consists of the following sections:
   * [`name`](#the-name-field) --- The name of the package.
   * [`version`](#the-version-field) --- The version of the package.
   * [`authors`](#the-authors-field) --- The authors of the package.
-  * [`edition`](#the-edition-field) --- The Rust edition.
-  * [`rust-version`](rust-version.md) --- The minimal supported Rust version.
-  * [`description`](#the-description-field) --- A description of the package.
+  * [`autobenches`](cargo-targets.md#target-auto-discovery) --- Disables bench auto discovery.
```

Comment:
For example, personally, `autobenches` is definitely not as important as other fields like `edition`, `rust-version`, and `description`. This might make the reading experience slightly worse for readers who go through it from top to bottom.

Reply:
FWIW, I agree with this, the ordering in the cargo docs seems better to me.